### PR TITLE
Issues/#1332 backport 1329

### PR DESCRIPTION
**Pull request by zcgandcomp**

### CODE REVIEW

File: powerauth-webflow/src/main/js/reducers/localeReducer.js [hunk 1]
```diff
@@ -29,7 +29,12 @@ export default function reducer(state = {locale: I18N_EN.locale, messages: I18N_
                 return {...state, locale: I18N_EN.locale, messages: I18N_EN.messages};
             } else if (action.locale === "cs") {
                 return {...state, locale: I18N_CS.locale, messages: I18N_CS.messages};
+            } else if (action.locale === "uk") {
+                return {...state, locale: I18N_UK.locale, messages: I18N_CS.messages};
```
Comment (romanstrobl) [inline powerauth-webflow/src/main/js/reducers/localeReducer.js:33]: `UK` locale should not map to `CS` messages.

File: powerauth-webflow/src/main/js/reducers/localeReducer.js [hunk 2]
```diff
@@ -29,7 +29,12 @@ export default function reducer(state = {locale: I18N_EN.locale, messages: I18N_
...
+            } else if (action.locale === "ro") {
+                return {...state, locale: I18N_RO.locale, messages: I18N_CS.messages};
```
Comment (romanstrobl) [inline powerauth-webflow/src/main/js/reducers/localeReducer.js:35]: `RO` locale should not map to `CS` messages.

File: powerauth-webflow/src/main/js/components/app.js [hunk 3]
```diff
@@ -68,6 +77,14 @@ export class App extends React.Component {
         document.cookie = "lang=" + lang + ";expires=" + d.toUTCString() + ";path=/";
     }
 
+    mapLanguageToCountry (locale) {
```
Comment (banterCZ) [inline powerauth-webflow/src/main/js/components/app.js:80]:
```suggestion
    mapLanguageToCountry(locale) {
```

File: powerauth-webflow/src/main/js/components/app.js [hunk 4]
```diff
@@ -68,6 +77,14 @@ export class App extends React.Component {
...
+        return this.languagesMapping.find(item => { return item.code == locale; }).country;
+    }
+
+    mapCountryToLanguage (flag) {
```
Comment (banterCZ) [inline powerauth-webflow/src/main/js/components/app.js:84]:
```suggestion
    mapCountryToLanguage(flag) {
```

**Review by banterCZ — APPROVED** [review]
Looks good.

**Pull request merged**